package.json

```diff
@@ -3,60 +3,60 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "react": "^16.9.0",
-    "react-dom": "^16.9.0",
-    "react-scripts": "3.1.1"
+    "react": "^16.12.0",
+    "react-dom": "^16.12.0",
+    "react-scripts": "^3.2.0"
   },
   "devDependencies": {
-    "@babel/core": "^7.5.5",
-    "@storybook/addon-actions": "^5.1.11",
-    "@storybook/addon-links": "^5.1.11",
-    "@storybook/addons": "^5.1.11",
-    "@storybook/cli": "^5.1.11",
-    "@storybook/react": "^5.1.11",
-    "@testing-library/react": "^9.1.3",
-    "@types/jest": "^24.0.18",
-    "@types/node": "^10.14.16",
-    "@types/react": "^16.9.2",
-    "@types/react-dom": "^16.9.0",
-    "@types/react-test-renderer": "^16.9.0",
-    "@types/storybook__addon-actions": "^3.4.3",
-    "@types/storybook__react": "^4.0.2",
-    "cypress": "^3.4.1",
-    "eslint-config-prettier": "^6.1.0",
-    "eslint-plugin-cypress": "^2.6.1",
-    "eslint-plugin-prettier": "^3.1.0",
-    "eslint-plugin-react": "^7.14.3",
-    "husky": "^1.3.1",
-    "hygen": "^4.0.8",
-    "loki": "^0.17.1",
+    "@babel/core": "^7.7.4",
+    "@storybook/addon-actions": "^5.2.6",
+    "@storybook/addon-centered": "^5.2.6",
+    "@storybook/addon-links": "^5.2.6",
+    "@storybook/addons": "^5.2.6",
+    "@storybook/cli": "^5.2.6",
+    "@storybook/react": "^5.2.6",
+    "@testing-library/react": "^9.3.2",
+    "@types/jest": "^24.0.23",
+    "@types/node": "^12.12.12",
+    "@types/react": "^16.9.12",
+    "@types/react-dom": "^16.9.4",
+    "cypress": "^3.6.1",
+    "eslint-config-prettier": "^6.7.0",
+    "eslint-plugin-cypress": "^2.7.0",
+    "eslint-plugin-prettier": "^3.1.1",
+    "eslint-plugin-react": "^7.16.0",
+    "husky": "^3.1.0",
+    "husky-add-issue-tracker": "0.0.9",
+    "hygen": "^5.0.3",
+    "loki": "^0.18.1",
     "npm-run-all": "^4.1.5",
-    "prettier": "^1.18.2",
-    "serve": "^11.1.0",
-    "source-map-explorer": "^2.0.1",
-    "start-server-and-test": "^1.10.0",
-    "typescript": "^3.5.3",
-    "wait-on": "^3.3.0"
+    "prettier": "^1.19.1",
+    "serve": "^11.2.0",
+    "source-map-explorer": "^2.1.1",
+    "start-server-and-test": "^1.10.6",
+    "typescript": "^3.7.2",
+    "wait-on": "^3.3.0",
+    "webpack": "^4.41.0"
   },
   "scripts": {
-    "start": "EXTEND_ESLINT=true SKIP_PREFLIGHT_CHECK=true react-scripts start",
-    "start:ci": "EXTEND_ESLINT=true SKIP_PREFLIGHT_CHECK=true npm-run-all build static",
-    "build": "EXTEND_ESLINT=true SKIP_PREFLIGHT_CHECK=true react-scripts build",
-    "test": "EXTEND_ESLINT=true SKIP_PREFLIGHT_CHECK=true react-scripts test --testMatch **/**/test.{ts,tsx}",
+    "start": "react-scripts start",
+    "start:ci": "npm-run-all build static",
+    "build": "react-scripts build",
+    "test": "react-scripts test --testMatch \"**/?(*.)+(spec|test).[jt]s?(x)\"",
```

Comment:
@roblafeve If we don't include `EXTEND_ESLINT=true` won't CRA just use its internal eslint config? [https://create-react-app.dev/docs/advanced-configuration](https://create-react-app.dev/docs/advanced-configuration)

Reply:
I think you're right. I will have to look into this a bit more. I know there's an additional "experimental" way to do this without mucking up the scripts, but I'm not using that at the moment. Will look into this further.

Reply:
@roblafeve What's this experimental way you're talking about? 👀

Reply:
@blaketarter https://create-react-app.dev/docs/setting-up-your-editor/#experimental-extending-the-eslint-config

Reply:
@blaketarter I've added `EXTEND_ESLINT=true` back to those scripts. I misunderstood what that bit in the docs was saying—they are still required.